semcore/card/src/Card.jsx

```diff
@@ -39,21 +40,37 @@ function Title(props) {
   const { styles, innerHint, Children, innerHintAriaLabel, hintAfterAriaLabel } = props;
   const hintAfter = props.hintAfter || props.hint;
   const STitle = Root;
-  const SIcon = InfoM;
+  const SIcon = ButtonLink;
   const STooltip = DescriptionTooltip;
   return sstyled(styles)(
     <>
       <STitle render={Text}>
         <Children />
         {innerHint && (
-          <STooltip ml={1} title={innerHint} aria-label={innerHintAriaLabel}>
-            <SIcon interactive />
+          <STooltip ml={1} title={innerHint}>
```

Comment:
can we merge tooltip trigger with the button (using `tag={}`)? 
otherwise we have 2 a11y audit issues:
* nested buttons (<button> inside an element with role=button)
* parent "button" isn't focusable

Reply:
fixed, now Tooltip has `Trigger` with tag `ButtonLink`